src/Daemon/BaseDaemon.cpp

```diff
@@ -1,3 +1,4 @@
+#include "Loggers/OwnFilteringChannel.h"
```

Comment:
This should go into the list of includes not at the first line.

Reply:
Whoops yeah, that must have been auto-generated by my IDE or something. Just fixed.